packages/libs/eda/src/lib/workspace/ImportAnalysis.tsx

```diff
@@ -44,15 +55,63 @@ export function ImportAnalysis({ analysisClient, analysisId }: Props) {
           };
           history.replace(newLocation);
         },
-        (error) =>
-          setError(error instanceof Error ? error.message : String(error))
+        (error) => {
+          const enforcedErrorObject =
+            error instanceof Error ? error : new Error(String(error));
+          const errorMessageList = enforcedErrorObject.message.split('\n');
+          try {
+            setError(JSON.parse(errorMessageList[1]));
+          } catch {
+            setError(enforcedErrorObject.message);
+          }
+        }
       );
   }, [analysisClient, history, analysisId]);
 
+  const handleClick = useCallback(
+    (event: React.MouseEvent<HTMLButtonElement>) => {
+      event.preventDefault();
+
+      if (user === undefined || user.isGuest) {
+        dispatch(showLoginForm());
+      }
+    },
+    [history, user, dispatch]
```

Comment:
> React Hook useCallback has an unnecessary dependency: 'history'. Either exclude it or remove the dependency array.eslintreact-hooks/exhaustive-deps)

I've got a handy ESLint plugin for VS Code that squiggles my hooks when they see something.

![image](https://user-images.githubusercontent.com/24446573/229598752-0dd996fa-1a0d-4a43-9da8-69b13f1bc561.png)